build/chopModuleProperties/ChopModulePropertiesTest.groovy

```diff
@@ -1,19 +1,53 @@
 import spock.lang.Specification
 
 import static org.jfrog.artifactory.client.ArtifactoryClient.create
+import org.apache.http.client.HttpResponseException
+import org.jfrog.artifactory.client.ArtifactoryRequest
+import org.jfrog.artifactory.client.impl.ArtifactoryRequestImpl
+
+import groovy.json.JsonSlurper
 
 class ChopModulePropertiesTest extends Specification {
```

Comment:
@theberbie since this and build.json files are present in /test folder, they must be removed from the plugin root folder.